src/seedu/addressbook/common/Messages.java

```diff
@@ -15,4 +15,5 @@
             "java seedu.addressbook.Main [STORAGE_FILE_PATH]";
     public static final String MESSAGE_WELCOME = "Welcome to your Address Book!";
     public static final String MESSAGE_USING_STORAGE_FILE = "Using storage file : %1$s";
+    public static final String MESSAGE_READ_ONLY = "Error - Storage file is read-only";
```

Comment:
`StorageOperationException` might occur because of other reasons (e.g. file in use), so we should suggest to the user what might have went wrong and how to fix it.